packages/components/src/components/drawer/drawer.jsx

```diff
@@ -0,0 +1,87 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class Drawer extends React.PureComponent {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            is_open: true,
+        };
+    }
+
+    toggleDrawer = () => {
```

Comment:
I think it would be better to create a `setVisibility` function so we can explicitly call a function to open or close the drawer. For example, when the user hits the run button we need to automatically open the drawer (to show trade summary), but what if the drawer is already open? Should we add logic to check if it's open before toggling, or would it be easier to call a `setVisibility` function?